fhir-persistence-schema/derby2.properties

```diff
@@ -0,0 +1,2 @@
+db.database=c:/users/mikeschroeder/openliberty-runtime-21.0.0.1/wlp/usr/servers/fhir-server/derby/study1
```

Comment:
This file can be removed - committed by mistake

Reply:
ok